.ci/integrationTestSelector.groovy

```diff
@@ -51,29 +52,80 @@ pipeline {
     /**
       launch integration tests.
     */
-    stage("Integration Tests"){
+    stage('Integration Tests'){
+      options { skipDefaultCheckout() }
       environment {
         TMPDIR = "${WORKSPACE}"
         ENABLE_ES_DUMP = "true"
         PATH = "${WORKSPACE}/${BASE_DIR}/.ci/scripts:${env.PATH}"
         APP = agentMapping.app(params.AGENT_INTEGRATION_TEST)
+        OPBEANS_APP = agentMapping.opbeansApp(params.AGENT_INTEGRATION_TEST)
       }
       when {
         expression {
           return (params.AGENT_INTEGRATION_TEST != 'All' &&
                   params.AGENT_INTEGRATION_TEST != 'Opbeans')
         }
       }
-      steps {
-        deleteDir()
-        unstash "source"
-        dir("${BASE_DIR}"){
-          sh(label: "Testing ${NAME} ${APP}", script: ".ci/scripts/agent.sh ${NAME} ${APP}")
+      parallel {
+        stage('Agent app') {
+          steps {
+            deleteDir()
+            unstash "source"
+            dir("${BASE_DIR}"){
+              sh(label: "Testing ${NAME} ${APP}", script: ".ci/scripts/agent.sh ${NAME} ${APP}")
+            }
+          }
+          post {
+            always {
+              wrappingup()
+            }
+          }
         }
-      }
-      post {
-        always {
-          wrappingup()
+        stage('Opbeans app') {
+          agent { label 'linux && immutable' }
+          options { skipDefaultCheckout() }
+          when {
+            expression { return (params.AGENT_INTEGRATION_TEST != 'RUM') }
+            beforeAgent true
+          }
+          steps {
+            deleteDir()
+            unstash "source"
+            dir("${BASE_DIR}"){
+              sh(label: "Testing ${NAME} ${OPBEANS_APP}", script: ".ci/scripts/opbeans-app.sh ${NAME} ${APP} ${OPBEANS_APP}")
+            }
+          }
+          post {
+            always {
+              wrappingup(isJunit: false)
+            }
+          }
+        }
+        stage('Opbeans RUM app') {
+          agent { label 'linux && immutable' }
+          options { skipDefaultCheckout() }
+          when {
+            expression { return (params.AGENT_INTEGRATION_TEST == 'RUM') }
+            beforeAgent true
+          }
+          steps {
+            deleteDir()
+            unstash "source"
+            dir('opbeans-frontend') {
```

Comment:
Opbeans-fronted is consumed here, in other words, there is no an opbeans-frontend image in the apm-its.

Before testing the RUM  PR, it's required to build the docker image from the source code and run the bump version script to change the agent version accordingly.